vendor/bat-native-ads/src/bat/ads/internal/security/crypto_util_unittest.cc

```diff
@@ -171,4 +173,24 @@ TEST(BatAdsSecurityUtilsTest, Sha256WithEmptyString) {
   EXPECT_TRUE(sha256.empty());
 }
 
+TEST(BatAdsSecurityCryptoUtilsTest, Encrypt) {
+  // Arrange
+  KeyPairInfo key_pair = GenerateBoxKeyPair();
```

Comment:
Can we have a separate test for the keypair generation itself (just for correct ouput)? Want to avoid future modifications that could tamper with this method.